tests/MetaPalettesTest.php

```diff
@@ -0,0 +1,348 @@
+<?php
+
```

Comment:
Fix header plz.

Reply:
Done in e57e911962f6c9815f2878d2024c6aa33e8908af.